integration/basic_test.go

```diff
@@ -380,9 +407,11 @@ func TestCollectionName(t *testing.T) {
 
 			assert.NoError(t, err)
 
-			names, err := collection.Database().ListCollectionNames(ctx, bson.D{})
+			newCollection := collection.Database().Collection(tc.collection)
+
+			// insert data to ensure collection can be updated.
+			_, err = newCollection.InsertOne(ctx, bson.D{})
 			require.NoError(t, err)
-			assert.Contains(t, names, tc.collection)
```

Comment:
Why did we change that? Was there a problem with the old test?

Reply:
Contains work fine. We changed how table name is mangled, I wanted to ensure that there is no error using mangled table name. I could have used `Find` but since collection was empty, I used Insert.